### DISCUSSION TITLE
Reference names but not generate documentation for third party libraries

### DISCUSSION THREAD

**Discussion by charlestang** [phpDocumentor/phpDocumentor#3656] [Q&A]
I used composer to introduce third party libraries into my projects, like web framework. In my project, lots of files are extends from base class in web framework. When I want to generate documentation of these sub classes, phpDocumentor will generate huge files.

Of cause I can use --ignore option of the command, but, all the parent name of these sub classes will be empty.

I want my documentation reference names in third party libraries but not include the documentation of them. I'll be very appreciated if you could tell me anything about achieving this.

**Comment by mvriel** [comment]
Hello @charlestang,

I am not quite sure what you would like to achieve. What kind of references are you looking for?
